<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeword</class>
 <widget class="QWidget" name="changeword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/static/images/bubu.ico</normaloff>:/static/images/bubu.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget  {
	border:1px solid black;
	border-radius:5px;
}</string>
  </property>
  <widget class="QWidget" name="titleWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>310</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#titleWidget{
background-color:rgb(22, 166, 243);
color:black;
border-radius:5px;
border:1px solid black;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover { 
 background-color: red; 
 color: rgb(255, 255, 255);
border: 1px solid black;
font-size:36px}
QPushButton {background-color:#E1E1E1;
color:black;
background-color: rgb(22, 166, 243); 
border-radius:5px;
border:0px solid black;
font-size:36px;}</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>单词本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>41</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton:hover { 
color:rgb(0, 0, 0);
border:0px solid black;
background-color:rgb(240,240,240);
}
QPushButton {
color:rgb(149, 149, 149);
border:0px solid black;
background-color:rgb(240,240,240);
}
</string>
   </property>
   <property name="text">
    <string>&lt; 返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="wordText">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>110</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>English</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="meanText">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>180</y>
     <width>171</width>
     <height>40</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>中文</string>
   </property>
  </widget>
  <widget class="QPushButton" name="findBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>修改/删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>1</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover { 
 background-color: red; 
 color: rgb(255, 255, 255);
border: 1px solid black;
	font: 36pt &quot;Microsoft YaHei UI&quot;;
}
QPushButton {background-color:#E1E1E1;
color:black;
background-color: rgb(22, 166, 243); 
border-radius:5px;
border:0px solid black;
	font: 36pt &quot;Microsoft YaHei UI&quot;;}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
